one-ui/src/main/java/com/marklogic/hub/oneui/util/SearchOptionBuilder.java

```diff
@@ -51,7 +51,7 @@ public String buildSearchOptions(String query, SearchQuery searchQuery) throws I
         // Setting search string if provided by user
         if (StringUtils.isNotEmpty(searchQuery.getQuery().getSearchStr())) {
             sb.append("<qtext>")
-                .append(StringEscapeUtils.escapeXml(searchQuery.getQuery().getSearchStr()))
+                .append(StringEscapeUtils.escapeXml10(searchQuery.getQuery().getSearchStr()))
```

Comment:
This is worth adding a unit test for - why escapeXml10 and not escapeXml? What scenario is this handling, such that it would break if we didn't call escapeXml10?